### PR TITLE
Filter commoncrawl warc files after exact timestamp, not only per year+month

### PR DESCRIPTION
**Pull request by lgov**
* commoncrawl_crawler.py: 
  (__get_remote_index): After filtering the warc files by year-month, also remove the ones from before warc_files_start_date.

### CODE REVIEW

File: tools/combine_json_outut.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# This tool combines all json files written by news-please in one new file, one line per
```
Comment (fhamborg) [inline tools/combine_json_outut.py:1]: this seems to be unrelated to the filtering mentioned in the PR's title. could you remove these additions?

**Review by fhamborg — CHANGES_REQUESTED** [review]

File: tools/combine_json_outut.py (hunk 1 above)
Comment (lgov) [reply tools/combine_json_outut.py]: I have removed b6881834094402e9ce23a6687e225c51e70d9f14 from my branch, it looks ok now.
